mcs/class/System/Test/System.Net.Sockets/SocketTest.cs

```diff
@@ -4319,6 +4319,67 @@ public void SendAsyncFile ()
 			}
 		}
 		
```

Comment:
try/catch/finally is not needed in your tests.
catch - failure will be detected by nunit
finally - you have using which does same